docs/manuals/source/TasksAndConcepts/BareosConsole.rst

```diff
@@ -286,10 +286,10 @@ nextvolume | nextvol
    Used in the list and llist commands. Takes no arguments.
 
 on
-   Takes no keyword.
+   Takes no argument.
 
 off
-   Takes no keyword.
+   Takes no argument.
```

Comment:
```suggestion
   Takes no arguments.
```

Reply:
I disagree, there is no clear pattern yet and singular makes more sense here, I think. Following are a few examples taken from the current documentation.

yes
Used in the restore command. Takes no argument.
listing
Permitted on the estimate command. Takes no argument.
done
Used in the restore command. Takes no argument.
media
Used in the list and llist commands. Takes no arguments.
pools
Used in the show, list, and llist commands. Takes no arguments.
select
Used in the restore command. Takes no argument.

Reply:
Hold on, there is a pattern. If the keyword is in plural form the description says it "takes no arguments". All the other cases say it "takes no argument". Why should we use the plural form in case of "on" and "off"? Please explain, thank you.